src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -13097,6 +13099,20 @@ namespace OutputReportTabular {
             OutputCompLoadSummary(facilityOutput, FacilityCoolCompLoadTables, FacilityHeatCompLoadTables, 0);
         }
 
+
+        // ZoneComponentLoadSummary: Now we convert and Display
+        if (displayZoneComponentLoadSummary) {
+            for (int iZone = 1; iZone <= NumOfZones; ++iZone) {
+                if (!ZoneEquipConfig(iZone).IsControlled) continue;
+                if (allocated(CalcFinalZoneSizing)) {
+                    LoadSummaryUnitConversion(ZoneCoolCompLoadTables(iZone));
+                    LoadSummaryUnitConversion(ZoneHeatCompLoadTables(iZone));
+
+                    OutputCompLoadSummary(zoneOuput, ZoneCoolCompLoadTables(iZone), ZoneHeatCompLoadTables(iZone), iZone);
+                }
+            }
+        }
+
```

Comment:
Thanks for structuring the code this way so that those time consuming routines are called for any IP unit